app/templates/admin/manage_categories.html

```diff
@@ -9,25 +9,25 @@
     <table>
         <thead>
             <tr>
-                <td>HTML Version</td>
-                <td>Human-readable Version</td>
+                <th>HTML Version</th>
+                <th>Human-readable Version</th>
                 <td></td>
                 <td></td>
```

Comment:
Should these two be <th> as well?

Reply:
The heads are hidden so those don't appear on small screens